Topic/mtlong2/asdplng 50 review profile rsyslog 2

mattlongt opened the pull request:
Reworked this profile as discussed with aloftus

mattlongt committed:
replaced manifests with the ones from Radiant, and added default data for new manifests

mattlongt committed:
changed class names

mattlongt committed:
added missing variable defaults

mattlongt committed:
removed one of the config rules for rsyslog client

mattlongt committed:
added test client rule to send logs to mlong-pup

mattlongt committed:
set relay to a test node for verification

mattlongt committed:
testing rsyslog relay with test data

mattlongt committed:
moved client.pp to init.pp, moved some data to control repo. removed duplications between relay and client

mattlongt committed:
changed class naming in relay.[[

mattlongt committed:
testing relay changes

mattlongt committed:
shuffled more data defaults around

mattlongt committed:
updated default values in hiera

mattlongt committed:
fixed pdk and yamllint warnings

mattlongt committed:
added a default ruleset so rsyslog will still start without extra configuration

andylytical reviewed (approved):
Looks good. One minor comment: remove the puppet strings docs in relay for the parameters that don't exist in the definition.

Also, don't forget to rebase before merging to main.

mattlongt committed:
moved puppet strings docks in init and relay for parameters that dont exist in the definitions anymore

mattlongt commented:
I removed the comments for definitions that don't exist in relay.pp anymore, and added a couple that moved from relay.pp to init.pp

No other changes were made outside of that in this latest commit, so I'll rebase and merge.
